docs/pages/blog/2019-developer-survey-results.md

```diff
@@ -7,10 +7,6 @@ card: false
 tags: ['Developer survey']
 ---
 
-# 2019 MUI Developer Survey results
-
-Olivier Tassinari, Matt Brookes. March 16, 2019.
```

Comment:
Could you rebase? This diff is already applied on *master*.

Reply:
That was fast! Yeah, will do it.

Reply:
When I use `git rebase master` I'm seeing _"Current branch details-polish is up to date."_. Am I doing something wrong?